contributors/devel/godep.md

```diff
@@ -90,6 +90,7 @@ rm -rf vendor
 ./hack/update-bazel.sh
 ./hack/update-godep-licenses.sh
 ./hack/update-staging-client-go.sh
+./hack/update-staging-godeps.sh
```

Comment:
@caesarxuchao well, I think if someone is following this document then that is always the case? If someone is not precisely following the document then there are probably lots of things that can go wrong and I don't think we have to document each of them? WDYT? I can certainly add a comment, just don't see the point.